### ISSUE TITLE
as a project owner, I'd like potential contributors to find my project

### ISSUE THREAD

**Issue by jbothma** [schalkventer/civic-tech-za#2]

**Comment by jbothma** [comment]
While civic tech projects go somewhat beyond open source projects, I think we can learn a lot from them in terms of being distributed, maintaining momentum (perhaps despite low velocity), and communicating project goals and current open tasks/needs.

@schalkneethling mentioned https://www.linuxfoundation.org/tools-managing-open-source-programs/ which at a glance focuses on tools for managing open source projects

We should also look at practises, not just tools.

**Comment by schalkventer** [comment]
@jbothma I agree. This is definitely something we can add to a resources list once we have it up and running. However, it might also be worthwhile discussing the above in relation to this specific repository - specifically regarding maintaining momentum and having iterations where we can determine what works and what doesn't.
